tests/Rector/Dynamic/FluentReplaceRector/Wrong/wrong.php.inc

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace SomeNamespace;
+
+class SomeClass
+{
+    public function someFunction()
+    {
+        foo();
+        return $this;
+    }
+
+    public function otherFunction()
+    {
+        bar();
+        return $this;
+    }
+}
+
+class ActionClass
+{
+    /**
+     * @var SomeClass
+     */
+    private $someClass;
+
+    public function someFunction()
+    {
+        $this->someClass = new SomeClass();
+
+        $this->someClass->someFunction()
+                        ->otherFunction();
+    }
+}
```

Comment:
Beautiful test case btw :+1:

Reply:
Thanks, I think in a good step to start.